contracts/test/forkMainnet/AMMWrapperWithPath/TradeUniswapV3.t.sol

```diff
@@ -187,41 +201,47 @@ contract TestAMMWrapperWithPathTradeUniswapV3 is TestAMMWrapperWithPath {
         BalanceSnapshot.Snapshot memory userTakerAsset = BalanceSnapshot.take(user, order.takerAssetAddr);
         BalanceSnapshot.Snapshot memory userMakerAsset = BalanceSnapshot.take(user, order.makerAssetAddr);
 
+        vm.prank(relayer, relayer);
         userProxy.toAMM(payload);
 
         userTakerAsset.assertChange(-int256(order.takerAssetAmount));
         userMakerAsset.assertChangeGt(int256(order.makerAssetAmount));
     }
 
     function testEmitSwappedEvent() public {
+        // this one was make compact in order to avoid stack too deep
+        // focusing on event fileds in this case
         uint256 feeFactor = 0;
```

Comment:
Should we include the case while the sender is the user himself instead of relayer, so we can check the feeFactor will be set to the defaultFeeFactor ?

Reply:
It's already covered in `CollectFee.t.sol`